lib/executor/constant_arrival_rate.go

```diff
@@ -196,9 +197,22 @@ func (car *ConstantArrivalRate) Init(ctx context.Context) error {
 	// with no work, as determined by their config's HasWork() method.
 	et, err := car.BaseExecutor.executionState.ExecutionTuple.GetNewExecutionTupleFromValue(car.config.MaxVUs.Int64)
 	car.et = et
+	start, offsets, lcd := et.GetStripedOffsets()
+	car.segIdx = lib.NewSegmentedIndex(start, lcd, offsets)
+
 	return err
 }
 
+// getNextGlobalIter advances and returns the next global iteration number for
+// this executor, taking into account the configured execution segment.
+// Unlike the local iteration number returned by getNextLocalIter(), this
+// iteration number will be unique across k6 instances.
+func (car *ConstantArrivalRate) getNextGlobalIter() uint64 {
+	res := car.segIdx.Next()
+	// iterations are 0-based
```

Comment:
```suggestion
	// iterations are indexed from 0
```
nit: this sounds better to me